## About Me
I am a postdoctoral scholar in the Department of Pathology at Stanford University. I obtained my PhD from Paul Sabatier Unniversity in Toulouse, France. There, I used RNA-sequencing and DNA-sequencing experiments to study the impact of variations on phenotypes. I worked on the characterization of genomic imprinting and RNA editing in chicken embryos. 
Strong of my experience in animal genomics, I wanted to transition from “feeding the world” to “healing the world” questions during my postdoctoral time. As a postdoctoral scholar in the [Montgomery lab](http://montgomerylab.stanford.edu/), I am working in understanding how genetic variants impact gene expression. During my first year at Stanford, I co-developed eigenMT, a multi-testing correction method for eQTL studies that accounts for linkage disequilibrium between variants. I then was involved in the development of CRISPR-X, a strategy to repurpose the somatic hypermutation machinery for protein engineering in situ.
More recently, I have been working on transcriptome samples from patients affected by rare disease with the will to identify disease causing genes in children where genome-based approaches were not successful.


## Publications

### Selected contributions
* **L. Fresard**, C Smail, K.S. Smith, N.M. Ferraro, N.A. Teran, K.D. Kernohan, D. Bonner, X. Li, S. Marwaha, Z. Zappala, B. Balliu, J.R. Davis, B. Liu, C.J. Prybol, J.N. Kohler, D.B. Zastrow, D.G. Fisk., M.E. Grove, J.M. Davidson, T. Hartley, R. Joshi, B.J. Strober, S. Utiramerur, Care4Rare Canada Consortium, Undiagnosed Diseases Network, L. Lind, E. Ingelsson, A. Battle, G. Bejerano, J.A. Bernstein, E.A. Ashley, K.M. Boycott, J.D. Merker, M.T. Wheeler, and S.B. Montgomery. Identication of rare-disease genes in diverse undiagnosed cases using whole blood transcriptome sequencing and large control cohorts. 2018. preprint [manuscript link](https://www.biorxiv.org/content/10.1101/408492v1)

* GTEx Consortium; Laboratory, Data Analysis and Coordinating Center (LDACC) - Analysis Working Group; Statistical Methods groups - Analysis Working Group; Enhancing GTEx (eGTEx) groups; NIH Common Fund; NIH/NCI; NIH/NHGRI; NIH/NIMH; NIH/NIDA; Biospecimen Collection Source Site - NDRI; Biospecimen Collection Source Site RPCI; Biospecimen Core Resource - VARI; Brain Bank Repository - University of Miami Brain Endowment Bank; Leidos Biomedical - Project Management; ELSI Study; Genome Browser Data Integration and Visualization - EBI; Genome Browser Data Integration and Visualization - UCSC Genomics Institute, University of California Santa Cruz; Lead analysts; Laboratory, Data Analysis and Coordinating Center (LDACC); NIH program management; Biospecimen collection:; Pathology; eQTL manuscript working group, Battle A, Brown CD, Engelhardt BE, Montgomery SB. Genetic effects on gene expression across human tissues. 2017. Nature. [manuscript link](https://www.nature.com/articles/nature24277)

* Hess GT, **L. Fresard**, K. Han, CH. Lee, A. Li, KA. Cimprich, SB. Montgomery, MC Bassik. Directed evolution using dCas9-targeted somatic hypermutation in mammalian cells. 2016. Nature Methods. [manuscript link](https://www.nature.com/articles/nmeth.4038)

* Davis JR¶, **L. Fresard**¶, DA. Knowles, M. Pala, CD. Bustamante, A. Battle, SB. Montgomery.  An Efficient Multiple-Testing Adjustment for eQTL Studies that Accounts for Linkage Disequilibrium between Variants. 2015. Americal Journal of Human Genetics. 
[manuscript link](https://www.cell.com/ajhg/fulltext/S0002-9297(15)00492-9).
[code](https://github.com/joed3/eigenMT)
¶ co-first

* **L. Fresard**, S. Leroux, B. Servin, D. Gourichon, P. Dehais, M.S. Cristobal, N. Marsaud, F. Vignoles, B. Bed'hom, J.-L. Coville, F. Hormozdiari, C. Beaumont, T. Zerjal, A. Vignal, M. Morisson, S. Lagarrigue, and F. Pitel. Transcriptome-wide investigation of genomic imprinting in chicken. 2014. Nucleic Acids Research.

### Complete list
You can find a list of my publications on [google scholar](https://scholar.google.com/citations?user=7keeHFQAAAAJ&hl=en)


## Conference presentation

* **Pathology department retreat**, Stanford ([talk](https://github.com/lfresard/talks-posters/blob/master/2018-10_pathology_retreat.pdf)), October 2018

* **American Society of Human Genetics Annual Meeting**, San Diego ([Poster](https://github.com/lfresard/talks-posters/blob/master/2018_ASHG_POSTER_lfresard_final.pdf)), October 2018

* **Biology of Genomes**, Cold Spring Harbor Laboratory (Poster), May 2018

* **Big Data in Biomedicine Conference**, Stanford, United States (Poster), May 2017

* **American Society of Human Genetics Annual Meeting**, Vancouver, Canada (Poster), October 2016

* **Plant and Animal Genome XXIII Conference** , San Diego, California, United States (Poster, Oral Presentation), January 2014

* **Epiconcept Workshop 2013** : Epigenetics for Improved Food Production: from Model to Practice}, Sant Feliu de Guíxols, Spain (Poster), October 2013


* **International Society for Animal Genetics 2012**, Cairns, Australia (Poster, Two Oral Presentations), July 2012


## Memberships/Volunteering work

*  Writer for a scientific podcast: [Goggles Optional](http://gogglesoptional.com/)  2015-2016
* Member of the [Stanford Penpal group](https://www.stanfordsciencepenpals.com/) 2016 - 2018
* Judge for [DNA day contest](https://www.ashg.org/education/dnaday.shtml) (ASHG) 2017
* Co-chair Programms Committee [AWIS Palo Alto](http://pa-awis.weebly.com/)
* Reviewer for: The Lancet, GigaScience, Gene


## Awards
* Volunteer of the Year 2018 - Association for Women In Science - Palo Alto Chapter, 2018
* Reviewers’ Choice Abstracts - ASHG, 2018
* Charles B. Carrington Memorial Award for outstanding poster presentation, 2017
* Stanford Center For Computational, Evolutionary and Human Genomics Postoctoral Fellowship, 2015


## Contact

* Mail:<li><a href="mailto:lfresard@stanford.edu">lfresard@stanford.edu</a></li>
* Twitter: @LaureFresard
* Linkedin: https://www.linkedin.com/in/lfresard/


